cdap-app-templates/cdap-program-report/cdap-program-report-base/src/main/java/co/cask/cdap/report/ReportGenerationSpark.java

```diff
@@ -111,9 +129,12 @@ protected void configure() {
     private static final String SAVED_FILE = "_SAVED";
     // report files will expire after 48 hours after they are generated
     private static final long VALID_DURATION_MILLIS = TimeUnit.DAYS.toMillis(2);
+    private static final String ENCRYPTION_ALGORITHM = "AES";
```

Comment:
Would be good to unify this constant between `SparkPersistRunRecordMain` and this file.